crates/sncast/src/starknet_commands/script.rs

```diff
@@ -37,18 +37,87 @@ use scarb_artifacts::ScarbCommand;
 use sncast::helpers::scarb_utils::{
     get_package_metadata, get_scarb_manifest, get_scarb_metadata_with_deps, CastConfig,
 };
-use sncast::response::structs::ScriptResponse;
+use sncast::response::print::{print_command_result, OutputFormat, OutputValue};
+use sncast::response::structs::{CommandResponse, ScriptResponse};
+use sncast::NumbersFormat;
 use starknet::accounts::Account;
 use starknet::core::types::{BlockId, BlockTag::Pending, FieldElement};
 use starknet::providers::jsonrpc::HttpTransport;
 use starknet::providers::JsonRpcClient;
 use tokio::runtime::Runtime;
 
+#[derive(Copy, Clone, Debug, Default, Eq, PartialEq, Ord, PartialOrd)]
+pub enum Verbosity {
+    /// Silence all script output except for errors
+    Quiet,
+
+    /// Default verbosity level
+    #[default]
+    Normal,
+}
+
+#[derive(Debug)]
+pub struct UI {
+    verbosity: Verbosity,
+    numbers_format: NumbersFormat,
+    output_format: OutputFormat,
+}
+
+impl UI {
+    pub fn new(is_quiet: bool, numbers_format: NumbersFormat, output_format: OutputFormat) -> Self {
+        let verbosity = if is_quiet {
+            Verbosity::Quiet
+        } else {
+            Verbosity::Normal
+        };
+
+        Self {
+            verbosity,
+            numbers_format,
+            output_format,
+        }
+    }
+
+    pub fn is_quiet(&self) -> bool {
+        self.verbosity < Verbosity::Normal
+    }
```

Comment:
It is necessary for quieting scarb, please mention it in the related issue which removes scarb build from declare and then remove it because of reasons mentioned above.

Reply:
removed this method